third_party/onnx/Makefile

```diff
@@ -0,0 +1,38 @@
+
```

Comment:
Just realised that we have another extra dependancy from `onnx` repo, which is the `onnx.proto` file. The files `onnx.pb.c` and `.h` that we have are generated based on that `.proto` file. Looks like this might be a good place to generate that files. I was doing it with `protoc` library:
```
protoc --c_out=. onnx.proto
```

Reply:
does this file also come with the onnx pip installation? just asking to determine if it makes sense to integrate a switch in the makefile to use the onnx repo or the installed version

Reply:
Yes, assuming you have `onnx` package installed, you can access the file with:

```python
import pkg_resources
path = "onnx.proto"
filepath = pkg_resources.resource_filename("onnx", path)
print(filepath)
```